opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -63,6 +65,20 @@ struct StackdriverOptions {
   std::unordered_map<std::string, google::api::MonitoredResource>
       per_metric_monitored_resource;
 
+  // Optional: per metric labels to add.
+  //
+  // These can overwrite view columns. If the value is empty, the column with
+  // the given key will be removed.
+  struct Label {
+    std::string key;
+    std::string value;
+  };
+  std::unordered_map<std::string, Label> per_metric_label;
+
+  // Optional: metrics for which the exporter should not
+  // CreateMetricDescriptor in Stackdriver.
+  std::unordered_set<std::string> per_metric_do_not_create_descriptor;
```

Comment:
Is this kind of removing?

Reply:
The intent (and I should improve the comment) is to export the metrics, but not send a CreateMetricDescriptor RPC because it will fail because the metric already exists (and we retry failures forever)

Reply:
We have a different logic here in go. It is called `builtinMetric`. See https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/master/stats.go#L201

Is that enough?

Reply:
That looks like the suggestion in https://github.com/census-instrumentation/opencensus-cpp/pull/373#issuecomment-526382831

@bianpengyuan if we use that logic to suppress both the RPC and the label, is that sufficient?

Reply:
Yes, that would be sufficient.

Reply:
Going with the `IsBuiltinMetric()` approach. Resolved.